lib/mongoid/association/referenced/has_many/enumerable.rb

```diff
@@ -209,9 +209,9 @@ def each
           # @since 2.1.0
           def empty?
             if _loaded?
-              in_memory.count == 0
+              in_memory.length == 0
```

Comment:
Can this simply call `in_memory.empty?` ?

Reply:
Yes, fixed.